rotkehlchen/tasks/assets.py

```diff
@@ -244,3 +260,74 @@ def update_owned_assets(user_db: DBHandler) -> None:
             'INSERT OR REPLACE INTO key_value_cache (name, value) VALUES (?, ?)',
             (DBCacheStatic.LAST_OWNED_ASSETS_UPDATE.value, str(ts_now())),
         )
+
+
+def update_underlying_assets(chains_aggregator: 'ChainsAggregator') -> None:
+    """Fetch the underlying defi assets and polulate `underlying_tokens_list` in globaldb"""
```

Comment:
this is only for aave v3 right? Use proper names in the func definition and also in the docstring. We can go generic once we need it but for now this function affects only to aave v3 assets